sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuTaskMetrics.scala

```diff
@@ -80,21 +80,35 @@ class NanoSecondAccumulator extends AccumulatorV2[jl.Long, NanoTime] {
 
 class GpuTaskMetrics extends Serializable {
   private val semWaitTimeNs = new NanoSecondAccumulator
-  private val spillBlockTimeNs = new NanoSecondAccumulator
-  private val readSpillTimeNs = new NanoSecondAccumulator
   private val retryCount = new LongAccumulator
   private val splitAndRetryCount = new LongAccumulator
   private val retryBlockTime = new NanoSecondAccumulator
   private val retryComputationTime = new NanoSecondAccumulator
 
+  // Spill
+  private val spillGpu2HostTimeNs = new NanoSecondAccumulator
+  private val spillGpu2DiskTimeNs = new NanoSecondAccumulator
+  private val spillHost2DiskTimeNs = new NanoSecondAccumulator
+
+  // Reverse Spill
+  private val readSpillHost2GpuTimeNs = new NanoSecondAccumulator
+  private val readSpillDisk2HostTimeNs = new NanoSecondAccumulator
```

Comment:
Nit: Not a fan of the use of numerals in the names, it saves a _single character_ so not really a big win for brevity.  I  think it's easier to read and clearer without them.